sdk/appcenter/src/test/java/com/microsoft/appcenter/IsRunningInAppCenterTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.appcenter;
+
+import android.os.Bundle;
```

Comment:
Unused imports again. Please use Android Studio to commit and check all options except TODO, that will avoid these comments.